### PR TITLE
Created CRUD for dynamic tasks

### PR DESCRIPTION
**Pull request by viniciusld**
#13 

@dbpolito

### CODE REVIEW

**Comment by dbpolito** [comment]
@wesleyhf please review and merge

File: database/migrations/2016_06_16_134621_create_tasks_table.php [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+        Schema::create('tasks', function (Blueprint $table) {
+            $table->increments('id');
+            $table->string('name');
+            $table->text('commands');
+            $table->timestamps();
+            $table->softDeletes();
+        });
+
+        Schema::table('deploys', function($table){
+            $table->integer('task_id')->unsigned()->nullable();;
```
Comment (wesleyhf) [inline database/migrations/2016_06_16_134621_create_tasks_table.php]: Double semicolon here.

File: resources/views/main.blade.php [hunk 2]
```diff
@@ -80,7 +93,12 @@
 
 			@else
 				<h3>Welcome </h3>
-				<p> Please, first register a server clicking <a href="{{ route('server.create')}}">here</a> </p>
+				@if($servers->count() <= 0)
```
Comment (wesleyhf) [inline resources/views/main.blade.php]: Change the if to `$servers->count() < 1`

File: resources/views/task/form.blade.php [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+                {{ method_field('PUT') }}
+            @else
+
+            <form action="{{ route('task.store') }}" method="post">
+
+            @endif
+
+                {{ csrf_field() }}
+                <div class="mdl-grid">
+                    <div class="mdl-cell mdl-cell--6-col mdl-cell--3-offset">
```
Comment (wesleyhf) [inline resources/views/task/form.blade.php]: Bad indentation. Remove the empty lines on form validation and indent all lines.

**Comment by wesleyhf** [comment]
Ready to merge, i don't have permission to merge.

File: app/Jobs/DeploymentQueueJob.php [hunk 4]
```diff
@@ -21,57 +19,48 @@ class DeploymentQueueJob extends Job implements ShouldQueue
     protected $deploy;
 
     /**
-     * Create a new job instance.
-     *
-     * @return void
-     */
+    * Create a new job instance.
+    *
```
Comment (dbpolito) [inline app/Jobs/DeploymentQueueJob.php]: bad indentation, docblockr should be like this: https://github.com/firework/deployer/pull/28/files#diff-204cb905769b13dff7f0a867415140a4R10

**Pull request merged**